packages/hub/readme.md

```diff
@@ -10,6 +10,20 @@ The simplest way to get this going on a mac is to install the [postgres app](htt
 
 You'll notice a set files with the `.env` prefix. The `.env` file contains environment variables that apply to all users across all environments. To add an environment variable specific to your local setup, add the variable to `.env.NODE_ENV.local` where NODE_ENV can be test, development, etc.
 
+#### Example local env file
+
+If you have `NODE_ENV=development`, you'd have a local file named `.env.development.local` with similar content to the following:
+
+```
+SERVER_URL=http://localhost:3002
```

Comment:
No need for duplicating `.env.development` settings. This is the logic for converting `.env` files to environment variables: https://github.com/statechannels/monorepo/blob/master/packages/devtools/src/config/env.ts#L11

`.local` files do not override other env files. They add env vars.

```suggestion
```